src/bootstrap/config.rs

```diff
@@ -1000,6 +1031,30 @@ impl Config {
             toml.merge(included_toml);
         }
 
+        for option in flags.options.iter() {
+            fn get_table(option: &str) -> Result<TomlConfig, toml::de::Error> {
+                toml::from_str(&option)
+                    .and_then(|table: toml::Value| TomlConfig::deserialize(table))
+            }
+
+            if let Ok(value) = get_table(option) {
+                toml.merge_replace(value);
+            } else {
+                // We want to be able to set string values without quotes,
+                // like in `configure.py`. Try adding quotes around the right hand side
+                if let Some((key, value)) = option.split_once("=") {
+                    if !value.contains('"') {
+                        if let Ok(value) = get_table(&format!(r#"{key}="{value}""#)) {
+                            toml.merge_replace(value);
+                            continue;
+                        }
+                    }
+                }
+                eprintln!("failed to parse override `{option}`");
```

Comment:
This error is not particularly helpful. Could you include the original error from `get_table`?
```
failed to parse override `build.codegen-backends=llvm`
```